addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -71,9 +71,51 @@ The Kodi thing supports the following channels:
 | shownotification        | String       | Shows the provided notification message on the screen |
 | input                   | String       | Allows to control Kodi. Valid values are: `Up`, `Down`, `Left`, `Right`, `Select`, `Back`, `Home`, `ContextMenu`, `Info`, `ShowCodec`, `ShowOSD` |
 | inputtext               | String       | This channel emulates a keyboard input |
-| systemcommand           | String       | This channel allows to send commands to shutdown/suspend/hibernate/reboot kodi |
+| systemcommand           | String       | This channel allows to send commands to `shutdown`, `suspend`, `hibernate`, `reboot` kodi |
 | mediatype               | String       | The media type of the current file. e.g. song or movie | 
 
+## Item Configuration
+demo.items
```

Comment:
please add empty lines before and after here, see https://www.eclipse.org/smarthome/documentation/development/bindings/docs.html